nuxeo-core/nuxeo-core-schema/src/test/resources/schema/facetSchema2.xsd

```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0"?>
+
+<xs:schema targetNamespace="http://www.nuxeo.org/ecm/schemas/relatedtext/"
+           xmlns:xs="http://www.w3.org/2001/XMLSchema" xmlns:nxs="http://www.nuxeo.org/ecm/schemas/relatedtext/">
+
+  <xs:simpleType name="clob">
+    <xs:restriction base="xs:string">
+      <xs:maxLength value="999999999" />
+    </xs:restriction>
+  </xs:simpleType>
+
+
+  <xs:element name="relatedtextresources" type="nxs:relatedtextresources" />
+
+  <xs:complexType name="relatedtextresources">
+    <xs:sequence>
+      <xs:element name="item" type="nxs:relatedtextresource" minOccurs="0"
+                  maxOccurs="unbounded" />
+    </xs:sequence>
+  </xs:complexType>
+
+  <xs:complexType name="relatedtextresource">
+    <xs:sequence>
+      <xs:element name="relatedtextid" type="xs:string" minOccurs="0" />
+      <xs:element name="relatedtext" type="nxs:clob" minOccurs="0" />
+    </xs:sequence>
+  </xs:complexType>
+
+</xs:schema>
```

Comment:
This whole file might not be needed, see https://github.com/nuxeo/nuxeo/pull/3666/commits/39d2703c7d8c2f1c51ce4c5c7b7e89c53cff8481#r368455261.
But, event if you want to declare a second facet, why such a complicated field/type?

Reply:
yes i agree that this one is little complicated, my idea was to add more complete schema with `blob`, `complex`...   to test that it will be correctly loaded :) -> but if  its enough complicated i will go to remove the second one or at least give more easy schema wdyth

Reply:
I think you can remove the second one. If I'm not mistaken, calling `SchemaManager#getFacet` and `CompositeType#getSchemaNames` doesn't actually load the facet/schemas but only fetched them from maps.